### ISSUE TITLE
Lathe manual

### ISSUE THREAD

**Issue by saras224** [bhattlab/lathe#22]
Hi @elimoss 

Do you have any manual where you have provided the explanations of the parameters that the lathe is using and also the steps that it is performing. For example when it says building DAG of jobs and then it gives the following list:

Job stats:
job                      count    min threads    max threads
---------------------  -------  -------------  -------------
align_bam                    3              5              5
all                          1              1              1
assemble_final               1              1              1
assemble_flye                2              5              5
bam_idx                      3              1              1
circularize_final            1              1              1
extract_bigtigs              1              1              1
faidx                        5              1              1
final                        1              1              1
merge                        1              1              1
misassemblies_correct        3              1              1
misassemblies_detect         3              1              1
nanoplot                     1              5              5
pilon_aggregate_vcf          1              1              1
pilon_consensus              1              1              1
pilon_ranges                 1              1              1
polish_final                 1              1              1
total                       30              1              5

I want to know the steps in detail for my better understanding. 

- When it is asking for the genome sze in the config file what exactly is that genome size? Is it of my metagenome data or that is the individual genome size that would be formed from the contigs after the assembly?

- Also when the assembly is completed what would be the final file that can be used for binning? 

- Basically, what I am asking is for a tutorial or a manual where information is given in steps and also how to do the downstream analysis after getting the assembly results?

Thanks in Advance
Saraswati Awasthi
